RetailCoder.VBE/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -431,10 +434,13 @@
             </ToolBar>
         </ToolBarTray>
 
-        <controls:EmptyUIRefresh Grid.Row="1" />
+        <TextBox Grid.Row="1" TextChanged="SearchBox_OnTextChanged" VerticalContentAlignment="Center" Name="SearchBox"></TextBox>
```

Comment:
I don't like handling the `TextChanged` event, but I didn't see a command to bind to. Let me know if you see something.

Reply:
Why isn't the tag self-closed?

Reply:
Getting much too used to HTML at work...